projects/fsastorefrontstyles/scss/base/comparison-table.scss

```diff
@@ -132,4 +131,35 @@
       border-left: none;
     }
   }
+  .ribbon {
+    position: relative;
+    width: calc(100% + 20px);
+    left: -10px;
+    top: 10px;
+    box-shadow: 0 0 0 2px var(--cx-color-primary),
+      0px 21px 5px -18px rgb(0 0 0 / 60%);
+    border-top: 1px dashed;
+    border-bottom: 1px dashed;
+    padding: 5px 0;
```

Comment:
We should start using `padding-block` and `padding-inline` instead of just padding => `padding-block: 5px;`